### ISSUE TITLE
Cannot connect to Deluge 1.3.12

### ISSUE THREAD

**Issue by ikus060** [erickok/transdroid#261]
Hello,

I recently update my Deluge server from 1.3.2 to 1.3.12. It seams to causes issue with Transdroid, So I update Transdroid to latest version. Still cannot connect to Deluge Server.

Here are the error log reported by Transdoid.
```
21 -- Sat Dec 05 23:12:53 EST 2015 -- 3 -- Error: java.net.SocketTimeoutException
22 -- Sat Dec 05 23:12:53 EST 2015 -- 3 -- ConnectionError exception: java.net.SocketTimeoutException
23 -- Sat Dec 05 23:14:12 EST 2015 -- 3 -- Error: java.net.SocketTimeoutException
24 -- Sat Dec 05 23:14:12 EST 2015 -- 3 -- ConnectionError exception: java.net.SocketTimeoutException
25 -- Sat Dec 05 23:14:27 EST 2015 -- 3 -- Error: java.net.SocketTimeoutException
26 -- Sat Dec 05 23:14:27 EST 2015 -- 3 -- ConnectionError exception: java.net.SocketTimeoutException
```

Using `tcpdump` I see some data getting into deluge-web. The following is the payload capture using tcpdump.
```
{"method":"web.update_ui","params":[["name","state","save_path","download_payload_rate","upload_payload_rate","num_peers","num_seeds","total_peers","total_seeds","eta","total_done","total_uploaded","total_size","progress","label","message","time_added","tracker_status"],[]],"id":2}
```

In deluge-web debug log, I also confirm reception of the query.
```
[DEBUG   ] 23:28:07 json_api:255 json-request: {"method":"web.update_ui","params":[["name","state","save_path","download_payload_rate","upload_payload_rate","num_peers","num_seeds","total_peers","total_seeds","eta","total_done","total_uploaded","total_size","progress","label","message","time_added","tracker_status"],[]],"id":2}
```

**Comment by saitoh183** [comment]
Same issue

**Comment by GrayaMax** [comment]
Same issue with Deluge 1.3.11 :(

**Comment by saitoh183** [comment]
Well mine started to work again

**Comment by erickok** [comment]
I have tested with 1.3.11 and 1.3.12 and they work just fine. Let me know if your issue persists.

**Comment by doublerebel** [comment]
I had a similar issue as this and #197.  I checked the transdroid error logs and saw that it suggested "Server time difference"? I then checked my server clock and saw that it had drifted about a minute off.  Fixed the clock and now all is fine.

That could explain why some people occasionally have this issue and then it goes away.  Anything running ntpd would fix itself and nobody would be the wiser.
